### PR TITLE
[Freeboxos] npe when updating inactive Player

### PR DESCRIPTION
**Pull request by clinique**
It can happen (when using webhooks) that a poll is sent when the player is not reachable.
This leads to an npe : 

```
2023-10-27 08:35:00.680 [WARN ] [mmon.WrappedScheduledExecutorService] - Scheduled runnable ended with an exception: 
java.lang.IllegalArgumentException: path is null
	at org.apache.cxf.jaxrs.impl.UriBuilderImpl.doPath(UriBuilderImpl.java:549) ~[?:?]
	at org.apache.cxf.jaxrs.impl.UriBuilderImpl.path(UriBuilderImpl.java:544) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.RestManager.buildUri(RestManager.java:59) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.RestManager.get(RestManager.java:73) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.RestManager.getSingle(RestManager.java:77) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.PlayerManager.getPlayerStatus(PlayerManager.java:214) ~[?:?]
	at org.openhab.binding.freeboxos.internal.handler.ActivePlayerHandler.internalPoll(ActivePlayerHandler.java:71) ~[?:?]
	at org.openhab.binding.freeboxos.internal.handler.ApiConsumerHandler.handleCommand(ApiConsumerHandler.java:150) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.WebSocketManager.lambda$1(WebSocketManager.java:112) ~[?:?]
	at java.util.HashMap$Values.forEach(HashMap.java:1065) ~[?:?]
	at org.openhab.binding.freeboxos.internal.api.rest.WebSocketManager.lambda$0(WebSocketManager.java:112) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) ~[?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]

```

### CODE REVIEW

**Review by lsiepel — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java [hunk 1]
```diff
@@ -59,32 +59,47 @@ void initializeProperties(Map<String, String> properties) throws FreeboxExceptio
...
+                    }
+                }
+                Configuration config = getManager(PlayerManager.class).getConfig(getClientId());
+                if (config != null) {
+                    uptime = checkUptimeAndFirmware(config.uptimeVal(), uptime, config.firmwareVersion());
+                } else {
+                    uptime = 0;
+                }
+            } else {
+                updateStatus(ThingStatus.OFFLINE);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java:91]: It seems to be already done at line 74

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java]: Yes, you're correct

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java [hunk 2]
```diff
@@ -31,7 +31,6 @@
 import org.openhab.core.thing.Thing;
 import org.openhab.core.thing.ThingStatus;
 import org.openhab.core.thing.binding.ThingHandlerService;
-import org.slf4j.Logger;
```
Comment (lolodomo) [inline bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java:34]: Import to be restored

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java]: Done

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**